flagsmith-jira-app/src/IssueFlagPanel.tsx

```diff
@@ -70,46 +70,77 @@ const IssueFlagForm = ({ features, featureIds, onAdd }: IssueFlagFormProps) => {
 };
 
 type IssueFlagTableProps = {
+  apiKey: string;
   projectUrl: string;
   environments: EnvironmentModel[];
   features: FeatureModel[];
-  flags: Record<string, FlagModel[]>;
   featureIds: string[];
   onRemove: (featureId: string) => Promise<void>;
   canEdit: boolean;
 };
 
 const IssueFlagTable = ({
```

Comment:
I think it would be good to add some documentation to this class. If I'm honest, I have no idea what's going. Is this a class to hold the information to render the features that have already been associated with the issue in JIRA? If so, my question on this PR would be why this has to change? I understood that we only needed to change the parameters that we send to the list features endpoint?

Reply:
This file was modified to optimize and accommodate changes in the endpoints:

Before: All flags and all feature states from each environment were returned, then they were searched based on the feature ID associated with the Jira ticket, and the information was displayed in the table.

Now: Only the last 50 flags are returned, and based on the IDs associated with Jira, the feature states of those flags associated with the ticket are obtained, and the information is displayed accordingly.

I will also add comments to explain some functionalities of the code that may not be very clear.